### PR TITLE
VSCodium: Add version 1.33.0

### PR DESCRIPTION
**Pull request by niheaven**
Free/Libre Open Source Software Binaries of VSCode

### CODE REVIEW

**Comment by niheaven** [comment]
Add vscodium-portable

File: bucket/vscodium-portable.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                "3673d7767da27418c4c7eea146043fa6a7c59df65199c79850e58367fd66fb56",
+                "9b55432e91e8534cc871d8d57f826761649d1ed7a3a5d7cabc3089fe5a6e147f"
+            ]
+        }
+    },
+    "notes": "Add VSCodium Portable as a context menu option by running: \"$dir\\vscodium-install-context.reg\"",
+    "bin": [
+        [
+            "bin\\vscodium.cmd",
+            "vscodium"
```
Comment (Ash258) [inline bucket/vscodium-portable.json]: Why this alias?

Comment (niheaven) [reply bucket/vscodium-portable.json]: Inherit from vscode.json, this is not needed, please wait.

Comment (r15ch13) [reply bucket/vscodium-portable.json]: I don't know why I added the alias with https://github.com/lukesampson/scoop-extras/commit/1ca87c4f39932f0030637eb86e82dede6e9a50da :grin:

**Pull request merged**